.github/workflows/build.yml

```diff
@@ -0,0 +1,267 @@
+#
+# This GitHub action builds Packer binaries, linux packages, 
+# and Docker images from source, and uploads them to GitHub artifacts. 
+# Note that artifacts available via GitHub Artifacts are not codesigned or notarized.
+#
+
+name: build
+
+on: [ workflow_dispatch, push, workflow_call ]
+
+env:
+  PKG_NAME: "packer"
+
```

Comment:
Do we any needs / use cases for adding any `GO_TAGS`? It can be left as an empty string in case any metadata can be appended Like its done in [CTS](https://github.com/hashicorp/consul-terraform-sync/blob/6049b348b13ad9ed599fb4ad33858f0a2fd722e6/.github/workflows/build.yml#L15) and the GO_TAG being used [here](https://github.com/hashicorp/consul-terraform-sync/blob/6049b348b13ad9ed599fb4ad33858f0a2fd722e6/.github/workflows/build.yml#L168)

Just a thought!

Reply:
That's a great idea! Adding this change in :)